recipe/build.sh

```diff
@@ -18,6 +12,13 @@ if [[ "${target_platform}" == *"linux"* ]]; then
     _target_sysroot=$(realpath $($CXX --print-sysroot))
     _target_rpcgen_bin=${_target_sysroot}/usr/bin/rpcgen
     _target_interpreter=$(realpath ${_target_sysroot}/$(patchelf --print-interpreter ${_target_rpcgen_bin}))
+    if [[ "${target_platform}" == "linux-s390x" ]]; then
+        # FIXME! the interpreter symlink found in the s390x sysroot is broken
+        # (literal 'ld64.so*' instead of 'ld64.so.1').
+        # this workaround assumes that there's exactly one ld-*.so library in the
+        # sysroot. if more than one or no library matches the glob, build will fail.
+        _target_interpreter=$(dirname "${_target_interpreter}")/ld-*.so
```

Comment:
it is never empty. it points to a file that doesn't exists.

Reply:
ah right, ok. I suppose by "build will fail" specifically it will fail a few lines down when `${_target_interpreter}` is called, rather than halfway through compilation

Reply:
which would be annoying to debug

Reply:
hmm, this is put in a wrapper which is called sometime later? So it might be good to check here if a library which matches the glob exists, rather than having the build fail when the rpcgen hack is run?

Reply:
> we need to fix the sysroot for s390x.

would you mind recording that in jira?

Reply:
yep, will do that tomorrow morning

Reply:
(also, just so I don't forget to mention, if that glob does not match, that would mean that the sysroot is seriously broken, even more than it is right now)